tests/track/test_track.py

```diff
@@ -0,0 +1,221 @@
+import shutil
+from filecmp import cmpfiles
+from pathlib import Path
+
+import pytest
+
+from OTVision.config import CONFIG
+from OTVision.helpers.files import get_files
+from OTVision.track.track import main as track
+
+SIGMA_L = CONFIG["TRACK"]["IOU"]["SIGMA_L"]
+SIGMA_H = CONFIG["TRACK"]["IOU"]["SIGMA_H"]
+SIGMA_IOU = CONFIG["TRACK"]["IOU"]["SIGMA_IOU"]
+T_MIN = CONFIG["TRACK"]["IOU"]["T_MIN"]
+T_MISS_MAX = CONFIG["TRACK"]["IOU"]["T_MISS_MAX"]
+
+
+def define_and_create_folders(
+    test_data_tmp_dir: Path, test_data_dir: Path, dir_name: str
+) -> tuple[Path, Path]:
+    test_track_dir = test_data_dir / "track" / dir_name
+    (test_data_tmp_dir / "track").mkdir(exist_ok=True)
+    test_track_tmp_dir = test_data_tmp_dir / "track" / dir_name
+    test_track_tmp_dir.mkdir(exist_ok=True)
+    return test_track_dir, test_track_tmp_dir
+
+
+def get_reference_data(test_track_dir: Path) -> tuple[list[Path], list[Path]]:
+    ref_detections_files = get_files(
```

Comment:
I suggest to not use OTVision code if they are not the subject of testing. For example, in the a future change int the `get_files` could lead to breaking tests relying on the method. 

We can either create our own helper methods to set up the test environment or my suggestion is to create pytest fixtures providing us in this case the ottrk file.

```python
import pytest
import shutil

@pytest.fixture
def track_test_data_dir(test_data_dir: Path) -> YieldFixture[Path]:
    track_data_dir = test_data_dir / "track"
    track_data_dir.mkdir(exist_ok=True)
    yield track_data_dir
    # teardown code
    shutil.rmtree(track_data_dir)



@pytest.fixture
def ottrk_file(test_track_data_dir)-> Path:
    return track_data_dir / "file.ottrk"

# In test_track
def test_track(ottrk_file: Path. some_args):
    do_something()
    # ...
```

Reply:
Thanks for the input, helped alot! Done :)